go.mod

```diff
@@ -1,85 +1,84 @@
 module github.com/openshift/managed-cluster-validating-webhooks
 
-go 1.23.0
+go 1.24.0
 
-toolchain go1.23.7
+toolchain go1.24.1
 
 require (
 	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/ghodss/yaml v1.0.1-0.20220118164431-d8423dcdf344
 	github.com/go-logr/logr v1.4.2
-	github.com/onsi/ginkgo/v2 v2.19.1
-	github.com/onsi/gomega v1.34.1
-	github.com/openshift/api v0.0.0-20240522145529-93d6bda14341
+	github.com/onsi/ginkgo/v2 v2.22.0
+	github.com/onsi/gomega v1.36.1
+	github.com/openshift/api v0.0.0-20250826155724-4a165b214722
 	github.com/openshift/cluster-logging-operator v0.0.0-20230328172346-05f4f8be54d5
 	github.com/openshift/hive/apis v0.0.0-20230327212335-7fd70848a6d5
 	github.com/openshift/operator-custom-metrics v0.5.1
 	github.com/openshift/osde2e-common v0.0.0-20231010150014-8a4449a371e6
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.55.1
-	github.com/prometheus/client_golang v1.20.4
-	gomodules.xyz/jsonpatch/v2 v2.3.0
-	k8s.io/api v0.29.0
-	k8s.io/apiextensions-apiserver v0.27.2
-	k8s.io/apimachinery v0.29.0
-	k8s.io/client-go v0.29.0
-	k8s.io/klog/v2 v2.120.1
-	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
-	sigs.k8s.io/controller-runtime v0.15.1
+	github.com/prometheus/client_golang v1.22.0
+	gomodules.xyz/jsonpatch/v2 v2.4.0
+	k8s.io/api v0.34.0
```

Comment:
Looks API jump from `v0.29` to `v0.34` is a big change? Looks it backward compatible for this codebase tho. LGTM.